crowbar_framework/app/models/node_object.rb

```diff
@@ -170,6 +170,10 @@ def self.all
     self.find nil
   end
 
+  def self.admin_node
+    all.map { |n| n if n.admin? }.first
```

Comment:
I think it is actually possible to do this more efficiently with a more targeted Chef search, but maybe it doesn't matter.